W-19473121-hsa-component-partial-kt (#322)
mulesoft/docs-hosting: hybrid-standalone/modules/ROOT/pages/creating-alerts-hybrid-apps.adoc

= Creating Alerts for Mule Apps Deployed to Hybrid Standalone

Configure proactive monitoring alerts for your Mule applications deployed to hybrid standalone instances to ensure optimal performance, detect issues early, and maintain application reliability across your distributed integration environment.

== Overview

Runtime Manager provides comprehensive alerting capabilities for monitoring your Mule applications deployed to hybrid standalone instances, including:

* Application performance monitoring (response time, throughput, resource usage)
* Business transaction tracking and error detection
* Custom application metrics and notifications
* Deployment lifecycle event monitoring
* Integration-specific health and availability monitoring

[NOTE]
====
Runtime Manager retains 30 days of alert history for audit and analysis.

Alerts retrigger when conditions continuously exceed specified thresholds, ensuring persistent notification of ongoing issues.
====

You can also create custom alerts for applications deployed to CloudHub and CloudHub 2.0.

> Custom alerts in CloudHub are triggered when your application sends notifications to the Runtime Manager console

Reviewer: ```suggestion
CloudHub  triggers custom alerts when your application sends notifications to the Runtime Manager console
```

using Anypoint Connector for CloudHub (CloudHub Connector). See xref:custom-application-alerts.adoc[Custom Application Alerts] for information.

> For applications deployed to CloudHub 2.0, custom alerts indicate whether an app was successfully deployed and whether it is behaving as expected. See xref:cloudhub-2::ch2-config-app-alerts.adoc[] for more information.

Reviewer: ```suggestion
For applications deployed to CloudHub 2.0, custom alerts indicate whether an app deploys successfully and whether it's behaving as expected. See xref:cloudhub-2::ch2-config-app-alerts.adoc[] for more information.
```

[NOTE]
For CloudHub 2.0, Alerts apply to applications only, not servers.

== Rate Limits on Alerts

> Anypoint Platform enforces rate limits that determine how often an alert is triggered and prevent excessive emails from being sent.

Reviewer: ```suggestion
Anypoint Platform enforces rate limits to control alert frequency and prevent excessive email notifications.
```

> At present, there is no defined limit to the number of Runtime Manager alerts.

Reviewer: ```suggestion
There's no defined limit to the number of Runtime Manager alerts.
```

== Create an Alert

Configure alerts to monitor your Mule applications with customizable thresholds and notification settings.

. In Runtime Manager, click *Alerts*.
+
** If no alerts exist, you see the initial setup page:
+
.The arrow shows the *Create your first alert* option.
image::alerts-no-alerts.png["Alerts page with no alerts set up"]
** For existing alert configurations, use the add button:
+
.The arrow shows the blue plus (*+*) icon on the *Alerts* page.
image::alerts-create-additional-alert.png["Alerts page with blue plus (+) icon"]

. Click the blue plus (*+*) icon to create a new alert.
+
image::alerts-create-alert.png["Create an Alert page"]

. Complete the required fields:

* *Name*
+
Unique and meaningful name for the alert
+
The *Name* is limited to 256 characters.
* *Severity level*
+
Severity of the alert: *Critical*, *Warning*, or *Info*
+
* *Source*
+
The source of the alert: applications
+
* *Application type*
+

> When the source is *Applications*, applications deployed to either CloudHub or local servers. Runtime Fabric application alerts are supported only using Anypoint Monitoring.

Reviewer: ```suggestion
When the source is *Applications*, applications deploy to either CloudHub or local servers. Runtime Fabric application alerts are supported only using Anypoint Monitoring.
```
